src/module/scoreboard.js

```diff
@@ -1,18 +1,24 @@
-const scorebox = () => {
+import getdata from './get.js';
+
+const scorebox = async () => {
+  const dataget = await getdata();
   const boardlist = document.querySelector('.board-listt');
   const board = document.createElement('ul');
   board.classList.add('score-box');
-  board.innerHTML = `
-  <li class="list" id="1"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="2"><p class="name">Farheen: </p><p> 99</p></li>
-  <li class="list" id="3"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="4"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="5"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="6"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="7"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="8"><p class="name">Afreen: </p><p>100</p></li>
-  <li class="list" id="9"><p class="name">Afreen: </p><p>100</p></li>
-`;
-  boardlist.appendChild(board);
+  boardlist.innerHTML = '';
+  for (let i = 0; i < dataget.length; i += 1) {
+    const li = document.createElement('li');
+    const name = document.createElement('p');
+    const score = document.createElement('p');
+    li.classList.add('list');
+    name.classList.add('name');
+    score.classList.add('score');
+    name.textContent = `${dataget[i].user}`;
+    score.textContent = `${dataget[i].score}`;
```

Comment:
- [ ] Hey, I noticed that dataget[i].score is returning [object Object]. This is because dataget[i].score is an object, not a string. To fix this, you can try using JSON.stringify(dataget[i].score) instead of ${dataget[i].score}. This will convert the object to a string and display it properly in the UI. Let me know if you have any questions!

See video below for more explanation:
https://www.loom.com/share/fd13fe38929d4b289e5efc4bd5204e16?sid=3462050b-36d6-4766-91e1-969885843c1e